css/index.css

```diff
@@ -1,3 +1,172 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* Desktop CSS here! */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
 
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+
+body .container {
+	margin: 0 auto;
+	max-width: 880px;
+}
+
+.navigation {
+	  display: flex;
+		flex-direction: row;
+		padding-top: 15px;
+		padding-bottom: 15px;
+		justify-content: space-between;
+		align-items: center;
+}
+
+nav a {
+	color: grey;
+	text-decoration: none;
+	height: inherit;
+	margin-right: 50px;
+}
+
+nav a:first-child {
+	padding-left: 0;
+}
+
+.logo {
+	float: right;
+	margin-left: 10px;
+}
+
+header {
+  margin-top: 45px;
+  margin-bottom: 50px;
+}
+
+.header {
+	display: inline-block;
+  vertical-align: top;
+  font-size: 40px;
```

Comment:
You might want to make this element a flexbox as well and set some flexbox properties - should be easier than dealing with `inline` and `block` level styling.